docs/source/markdown/podman-auto-update.1.md

```diff
@@ -0,0 +1,46 @@
+% podman-auto-update(1)
+
+## NAME
+podman\-auto-update - Auto update containers according to their auto-update policy
```

Comment:
do you need a '\' before `-update`?

Reply:
Good catch. I just tested, the backslash isn't needed :)